### DISCUSSION TITLE
Ομαδική απάντηση στα μέρη Β και Γ και εφαρμογή λογοκλοπής

### DISCUSSION THREAD

**Discussion by mxidakis** [sdy60/syllabus#175] [Q&A] [answered]
Η απάντηση στο Μέρος Β και Γ της ΓΕ2 θα είναι η ίδια για τα μέλη της κάθε ομάδας;
Δεν θα παρουσιαστεί πρόβλημα από το turnitin στη πλατφόρμα υποβολής της εργασίας;

**Comment by epidrome** [comment, marked as answer]
Ναι, θα είναι ίδια, η εφαρμογή λογοκλοπής ήδη βγάζει μεγάλα ποσοστά αφού η εκφώνηση είναι μερικές φορές μεγαλύτερη από την απάντηση, ειδικά όταν έχουμε αρκετές οθόνες αντί για κείμενο.
